tests/unit/ssg-module/data/file_owner_grub2_cfg.yml

```diff
@@ -0,0 +1,30 @@
+# File processed by the build system, i.e. retrieved from the build/rhel7/rules/file_owner_grub2_cfg.yml
+#
+description: 'The file <tt>/boot/grub2/grub.cfg</tt> should
+
+  be owned by the <tt>root</tt> user to prevent destruction
+
```

Comment:
Why are these empty lines there?

Reply:
It has been taken straight from the build, so this corresponds to the produced content.